plugins/woocommerce/src/Internal/Admin/BlockTemplates/README.md

```diff
@@ -0,0 +1,3 @@
+# Block Template API Documentation
+
+> ⚠️ **Notice:** This documentation is currently a **work in progress**. While it's open to the public for transparency and collaboration, please be aware that some sections might be incomplete or subject to change. We appreciate your patience and welcome any contributions!
```

Comment:
I'd leave these placeholder files out.

I'll try to get the actual docs for the template API up for review by the end of the week (or early next week worst case).